### PR TITLE
OCPBUGS-10765: Remove oldest ovn acl log files when file limit exceeded

### PR DESCRIPTION
**Pull request by jluhrsen**
adds new api config for max total storage with a default of 5GB. once the /var/log/ovn/ folder has exceeded that limit, the oldest files will be removed (one at a time) to keep within the limit.

### CODE REVIEW

File: pkg/network/ovn_kubernetes.go [hunk 1]
```diff
@@ -235,7 +235,8 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data.Data["IPFIXCacheActiveTimeout"] = ""
 	data.Data["IPFIXSampling"] = ""
 	data.Data["OVNPolicyAuditRateLimit"] = c.PolicyAuditConfig.RateLimit
-	data.Data["OVNPolicyAuditMaxFileSize"] = c.PolicyAuditConfig.MaxFileSize
+	data.Data["OVNPolicyAuditMaxLogFiles"] = c.PolicyAuditConfig.MaxLogFiles
+	data.Data["OVNPolicyAuditMaxTotalStorage"] = c.PolicyAuditConfig.MaxTotalStorage
```
Comment (kyrtapz) [inline pkg/network/ovn_kubernetes.go:239]:
```suggestion
	data.Data["OVNPolicyAuditMaxFileSize"] = c.PolicyAuditConfig.MaxFileSize
```

File: bindata/network/ovn-kubernetes/managed/ovnkube-node.yaml [hunk 2]
```diff
@@ -249,6 +250,13 @@ spec:
               CONTROLLERPID=$(cat /run/ovn/ovn-controller.pid)
             fi
 
+            # Ensure total number of log files does not exceed the maximum configured from OVNPolicyAuditMaxLogFiles
+            num_files=$(ls -1 $log_dir/acl-audit-log* 2>/dev/null | wc -l)
+            if [ "$num_files" -gt "$max_files" ]; then
```
Comment (kyrtapz) [inline bindata/network/ovn-kubernetes/managed/ovnkube-node.yaml:255]:
```suggestion
            if [ "$num_files" -gt "$MAXLOGFILES" ]; then
```

**Comment by jluhrsen** [comment]
just FYI that this is waiting on the completion of the [CNO rebase to 1.27](https://issues.redhat.com/browse/SDN-3901) which will bring in the new openshift/api changes we need.

**Pull request closed without merging**